<?xml version="1.0"?>
<launch>

<arg name="do_project_vehicle_lane" default="true"/>
<arg name="do_project_laser_scans" default="true"/>

<group ns="Operator/Projection">
<node name="VideoProjection" pkg="tod_projection" type="OperatorVideoProjection" output="screen">
    <param name="do_project_vehicle_lane" value="$(arg do_project_vehicle_lane)"/>
    <param name="do_project_laser_scans" value="$(arg do_project_laser_scans)"/>
</node>

<node name="LaneProjection" pkg="tod_projection" type="OperatorLaneProjection" output="screen"/>
</group>

</launch>
